### PR TITLE
More small changes to docs

### PR DESCRIPTION
**Pull request by alexwlchan**
* En dashes instead of hyphens
* Spelling fixes
* Write out small numbers
* Spacing and formatting tweaks
* Line wrapping

Assigning this to 0.21 because the changes are all very minor, and so there's no urgency to get them into Monday's 0.20 release.

### CODE REVIEW

File: docs/source/configuration.rst [hunk 1]
```diff
@@ -16,9 +16,9 @@ Configuring Calico
 ==================
 
 This page describes how to configure Calico. We first describe the
-configuration of the core Calico component - Felix -
+configuration of the core Calico component -- Felix --
 because this is needed, and configured similarly, regardless of the
-surrounding environment (OpenStack, Docker, or whatever). Then,
+surrounding environment (OpenStack, Docker, or whatever.) Then,
```
Comment (ed-harrison) [inline docs/source/configuration.rst]: I thought the rule was that you should be able to take the part in parentheses away without breaking the sentence - i.e. in this case, taking away the parentheses section will take out the full stop, which is wrong.

Comment (lxpollitt) [inline docs/source/configuration.rst]: +1 for @ed-harrison's comment.  :-)

Comment (alexwlchan) [inline docs/source/configuration.rst]: I misread this; for some reason I thought it was a sentence entirely in parentheses. Oops; fix coming.

**Comment by ed-harrison** [comment]
LGTM

**Pull request merged**